Automattic/woocommerce-payments
# Add/better error message for minimum required amount

Fixes #701 

#### Changes proposed in this Pull Request

* Localized error message if order amount is too low to process, based on payments server response.

#### Testing instructions

 * On a connected WooCommerce store create a product with a price below 0.5 USD
 * Put it into cart and try to purchase through WCPay
 * Error message should say:`Sorry, the minimum allowed order total is $0.50 to use this payment method`
  
<img width="800" alt="image" src="https://user-images.githubusercontent.com/683297/106288936-f30fd300-6248-11eb-8f13-b3ed873b9d05.png">
-------------------
